lib/Dialect/LTL/LTLTypes.cpp

```diff
@@ -0,0 +1,24 @@
+#include "circt/Dialect/LTL/LTLTypes.h"
+#include "circt/Dialect/LTL/LTLOps.h"
+
+using namespace circt;
+using namespace ltl;
+
+bool circt::ltl::isClocked(mlir::Type type) {
+  return isa<ClockedPropertyType>(type) || isa<ClockedSequenceType>(type) ||
+         isa<ClockedDisabledPropertyType>(type);
```

Comment:
FYI you can pass multiple classes to `isa<>`, e.g:
```suggestion
  return isa<ClockedPropertyType, ClockedSequenceType, ClockedDisabledPropertyType>(type);
```